### PR TITLE
Fix create extension mojo for Camel 4.5.x

### PR DESCRIPTION
**Pull request by jamesnetherton**

### CODE REVIEW

File: common/src/main/java/org/l2x6/cq/common/CqCatalog.java [hunk 1]
```diff
@@ -82,6 +83,17 @@ public String getArtifactId() {
         }
     }
 
+    static final Predicate<Kind> SUPPORTED_CATALOG_KING_FILTER = kind -> {
```
Comment (ppalaga) [inline common/src/main/java/org/l2x6/cq/common/CqCatalog.java:86]:
```suggestion
    static final Predicate<Kind> SUPPORTED_CATALOG_KIND_FILTER = kind -> {
```

Catalog is the king!

**Review by ppalaga — APPROVED** [review]
Thanks @jamesnetherton!

File: common/src/main/java/org/l2x6/cq/common/CqCatalog.java (hunk 1 above)
Comment (jamesnetherton) [reply common/src/main/java/org/l2x6/cq/common/CqCatalog.java:86]: Heh - nice catch 🤴 

I'll fix it.

**Comment by jamesnetherton** [comment]
I'll release 4.7.1 shortly.

**Pull request merged**